Taskfile.yml

```diff
@@ -52,7 +55,7 @@ tasks:
     cmds:
       - go build ./cmd/arduino-app-cli # needed for e2e tests
       - task: generate
-      - go test ./internal/... ./cmd/... -v -race {{ .CLI_ARGS }}
+      - go test $(go list ./internal/... ./cmd/... | grep -v internal/e2e/updatetest) -v -race {{ .CLI_ARGS }} # exclude deb package update tests
```

Comment:
I would propose this instead

```
  test:internal:
    cmds:
      - go test $(go list ./internal/... ./cmd/... | grep -v internal/e2e) -v -race {{ .CLI_ARGS }} # exclude e2e tests

  test:e2e:
    cmds:
      - go build ./cmd/arduino-app-cli
      - task: generate
      - go test ./internal/e2e/... --count=1 -v {{ .CLI_ARGS }}
```

one task command for all the test and one for all the e2e including you new tests